pal/audio/minigame/player-web.ts

```diff
@@ -80,6 +80,12 @@ export class AudioPlayerWeb implements OperationQueueable {
      */
     public _operationQueue: OperationInfo[] = [];
 
+    public getSampleRate (): number {
+        return this._audioBuffer.sampleRate;
+    }
+    public getBuffer (channelID: number): Float32Array {
+        return this._audioBuffer.getChannelData(channelID);
+    }
     constructor (audioBuffer: AudioBuffer, url: string) {
         this._audioBuffer = audioBuffer;
```

Comment:
also we need to handle the  `minigame/player.ts` and `minigame/player-minigame.ts`
```ts
// player-minigame.ts
    get sampleRate (): number {
        return 0;
    }

    public getBufferAtChannel (channelID: number): Float32Array | undefined {
        return undefined;
    }
```